Corona-Warn-App/src/main/java/de/rki/coronawarnapp/contactdiary/ui/day/sheets/person/ContactDiaryPersonBottomSheetDialogViewModel.kt

```diff
@@ -0,0 +1,28 @@
+package de.rki.coronawarnapp.contactdiary.ui.day.sheets.person
+
+import com.squareup.inject.assisted.AssistedInject
+import de.rki.coronawarnapp.contactdiary.storage.ContactDiaryRepository
+import de.rki.coronawarnapp.ui.SingleLiveEvent
+import de.rki.coronawarnapp.util.coroutine.DispatcherProvider
+import de.rki.coronawarnapp.util.viewmodel.CWAViewModel
+import de.rki.coronawarnapp.util.viewmodel.SimpleCWAViewModelFactory
+
+class ContactDiaryPersonBottomSheetDialogViewModel @AssistedInject constructor(
+    dispatcherProvider: DispatcherProvider,
+    private val contactDiaryRepository: ContactDiaryRepository
+) : CWAViewModel(dispatcherProvider = dispatcherProvider) {
```

Comment:
How would we be able to edit an existing person?

Reply:
I would've actually built a completely different bottom sheet for that as these are in the day packages and the desings do not define any edit functionality

Reply:
Hm, I would reuse it, especially when there are later more input fields we would have a lot of duplicate code.
The only different between both would be save vs overwrite, and a different button?

Reply:
@kolyaopahle I can also adapt it to be reusable in my branch where I would also reuse it.

Reply:
Sure, would appreciate the lower workload 😅

Reply:
My idea was to either launch the bottom sheets with an argument (item ID) or without. With item ID as argument it is in "edit mode", without it is in "create new" mode.